Svc/DpCatalog/test/ut/DpCatalogTester.cpp

```diff
@@ -93,6 +94,7 @@ namespace Svc {
             } else {
                 ASSERT_TRUE(res != nullptr);
             }
+            //printf("CE: %u\n",entry);
```

Comment:
More dead code.

Reply:
This is UT code, so I like to keep them around in case I need to dump it again. It's not flight code, so does it apply?